lib/ExtUtils/Liblist/Kid.pm

```diff
@@ -88,6 +90,7 @@ sub _unix_os2_ext {
                 $thislib = $self->catdir( $pwd, $thislib );
             }
             push( @searchpath, $thislib );
+            $thislib = qq{"$thislib"} if $thislib =~ / /; # protect spaces if there
```

Comment:
I don't think this is correct quoting, it handles spaces but what about quotes or backslashes?

Reply:
I agree with your point. Are you aware of a converse to `Text::ParseWords` that reverse?